### ISSUE TITLE
Critical dependency warning: localforage.js

### ISSUE THREAD

**Issue by LarryEitel** [codecasts/spa-starter-kit#113]
When running `yarn run dev`:

```
Child html-webpack-plugin for "index.html":
         Asset     Size  Chunks       Chunk Names
    index.html  1.47 MB       0
webpack: bundle is now VALID.
webpack: bundle is now INVALID.
webpack building...
webpack built 6207064b6b09e040d5fd in 285ms
Hash: 6207064b6b09e040d5fd
Version: webpack 1.14.0
Time: 285ms
                 Asset       Size  Chunks             Chunk Names
      0.auth-bundle.js    35.3 kB       0             auth-bundle
     1.parks-bundle.js     582 kB       1             parks-bundle
  2.products-bundle.js      57 kB       2             products-bundle
3.categories-bundle.js    54.2 kB       3             categories-bundle
                app.js    6.21 MB       4             app
 5.dashboard-bundle.js      14 kB       5             dashboard-bundle
            index.html  342 bytes          [emitted]

WARNING in ./~/localforage/dist/localforage.js
Critical dependencies:
7:484-491 This seems to be a pre-built javascript file. Though this is possible, it's not recommended. Try to require the original source to get better results.
 @ ./~/localforage/dist/localforage.js 7:484-491
```

**Comment by lednhatkhanh** [comment]
I got the same warning

**Comment by lednhatkhanh** [comment]
@LarryEitel Are you using windows? I created a pull request for this error here #115, please help me test the result.

**Comment by LarryEitel** [comment]
Thank you the warning went away.

**Comment by vinicius73** [comment]
closed on #115
